components/ThePlayground.client.vue

```diff
@@ -3,95 +3,31 @@
 import { Pane, Splitpanes } from 'splitpanes'
 
 const iframe = ref<HTMLIFrameElement>()
-const wcUrl = ref<string>()
 
-type Status = 'init' | 'mount' | 'install' | 'start' | 'ready' | 'error'
-
-const status = ref<Status>('init')
-const error = shallowRef<{ message: string }>()
 const isDragging = usePanelDragging()
+const panelSizeEditor = usePanelCookie('nuxt-playground-panel-editor', 30)
+const panelSizeFrame = usePanelCookie('nuxt-playground-panel-frame', 30)
 
-const panelSizeEditor = useLocalStorage('nuxt-playground-panel-editor', 30)
-const panelSizeFrame = useLocalStorage('nuxt-playground-panel-frame', 30)
-
-const stream = ref<ReadableStream>()
-
-async function startDevServer() {
-  const tree = globFilesToWebContainerFs(
-    '../templates/basic/',
-    import.meta.glob([
-      '../templates/basic/**/*.*',
-      '../templates/basic/**/.*',
-      '!../.DS_Store',
-      '!**/node_modules/**',
-    ], {
-      as: 'raw',
-      eager: true,
-    }),
-  )
-
-  console.log({ tree })
-
-  const wc = await useWebContainer()
-
-  wc.on('server-ready', (port, url) => {
-    // Nuxt listen to multiple ports, and 'server-ready' is emitted for each of them
-    // We need the main one
-    if (port === 3000) {
-      status.value = 'ready'
-      wcUrl.value = url
-    }
-  })
```

Comment:
You actually reverted quite a few changes. Let's not refactor them right now, as we will have quite some different needs to cover afterwards. I'll take over and revert some changes.

Reply:
Yes I know that not everything has been changed. I wanted to make a more readable composable for web container